src/app/MainMenu.js

```diff
@@ -0,0 +1,195 @@
+const MENU = {
+            mainPage: {name: 'Main Page', link: ''},
+            randomRecipe: {name: 'Random Recipe', link: ''},
+            nutritionGame: {name: 'Nutrition Game', link: ''},
+            calculator: {name: 'Calculator', link: ''},
+            shoppingList: {name: 'Shopping List', link: './list.html'}
+          }
+
+
+export const MainMenu = (activePage) => {
```

Comment:
You have Kinga's changes, watch out for such a situation, you can push code to production which is not ready for it